### PR TITLE
Feat cards

### PR DESCRIPTION
**Pull request by Samarth-5109**
Card
Cards with badges
Cards with dismiss
Cards with text overlay
Text only cards
Vertical/Horizontal cards
Cards with shadow

wuipLff4Wu

### CODE REVIEW

**Review by GuptaEesh — COMMENTED** [review]
The code looks fine. The only negatives are:
- Very minimal use of CSS variables.
- You can use rem for values >10px or values > 1rem.

File: components/cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    margin-top: 3rem;
+}
+
+.basic-card{
+    display: flex;
+    flex-direction: column;
+    width: 20rem;
+    margin: 1rem;
+    font-family: 'Roboto', sans-serif;
+    box-shadow:20px 20px 50px grey;
```
Comment (Aditi0505) [inline components/cards/cards.css:15]: Suggestion: use hex code and can use a css variable for color.

File: components/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    display: flex;
+    justify-content: space-between;
+    padding: 10px;
+}
+
+.action-btn{
+    padding:8px;
+    background-color: #456568;
+    border: 0 solid transparent;
+    color: whitesmoke;
```
Comment (Aditi0505) [inline components/cards/cards.css:54]: Suggestion: Can use Hex value for consistency.

File: components/cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    border-radius:5px;
+}
+
+.card-icons{
+    padding: 8px;
+}
+
+.action-btn:hover, .card-icons:hover{
+    background-color: whitesmoke;
```
Comment (Aditi0505) [inline components/cards/cards.css:63]: Same as above

File: components/cards/cards.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    color:#456568;
+}
+
+.horizontalCard{
+    position: relative;
+/*     height:130px; */
```
Comment (Aditi0505) [inline components/cards/cards.css:69]: Can remove commented code if unnecessary.

File: components/cards/cards.html [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+</head>
+<body>
+    <header class="nav-container">
+        <div class="branding">
+           <a href="index.html"><img class="brand_image"src="https://cdn-0.emojis.wiki/emoji-pics/google/nazar-amulet-google.png" alt="brand-image"></a> 
+            <span class="brand_name">NETRA UI</span>
+        </div>
+        <nav class="navbar">
+            <ul class="nav-items">
+                <!-- <li><a href="">Home</a></li> -->
```
Comment (Aditi0505) [inline components/cards/cards.html:19]: Can remove commented code if not needed.

File: components/cards/cards.html [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+                <li class="list-items item-1"><a href="" class="doc">Documentation</a></li>
```
Comment (Aditi0505) [inline components/cards/cards.html:20]: Can use placeholder in href attribute (# ) if it is not suppose to re-direct somewhere.

File: components/cards/cards.html [hunk 7]
```diff
@@ -0,0 +1,277 @@
...
+                <li class="list-item"><a href="avatar.html">Avatars</a></li>
+                <li class="list-item"><a href="badges.html">Badges</a></li>
+                <li class="list-item"><a href="button.html">Buttons</a></li>
+                <li class="list-item"><a href="cards.html">Cards</a></li>
+                <li class="list-item"><a href="images.html">Image</a></li>
+                <li class="list-item"><a href="">Inputs</a></li>
+                <li class="list-item"><a href="">Lists</a></li>
+                <li class="list-item"><a href="">Navigation</a></li>
+                <li class="list-item"><a href="">Slider</a></li>
+                <li class="list-item"><a href="">Toast</a></li>
```
Comment (Aditi0505) [inline components/cards/cards.html:40]: Here too, href is not pointing to anything. Are you planning to add references at the end? Let me know your thoughts on this.

File: components/cards/cards.html [hunk 8]
```diff
@@ -0,0 +1,277 @@
...
+                    <p class="about">Card are used to show user related data collectively, like product details.</p>
+                </div>
+            </div>
+           
+            <div class="type-wrapper">
+                <div class="component-type">
+                    <span class="type-title"> Vertical Card</span>
+                </div> 
+                <div class="about-type">
+                    <!-- <p class="about">Use the class <span class="required">required-field</span>  if you want the field to be mandatory, with required attribute in the input tag. </p>  -->
```
Comment (Aditi0505) [inline components/cards/cards.html:62]: Can remove commented code.

File: components/cards/cards.html [hunk 9]
```diff
@@ -0,0 +1,277 @@
...
+                </div>
+                <img class="code-Ss" src="../../images/verticalCard.png" alt="code-shot">
+            </div>
+            <div class="types-container">
+                <div class="basic-card">
+                    <img src="https://m.media-amazon.com/images/I/718FX4+xmCL._AC_SS450_.jpg" alt="">
```
Comment (Aditi0505) [inline components/cards/cards.html:68]: Can provide alt description for accessibility.

File: components/cards/cards.html [hunk 10]
```diff
@@ -0,0 +1,277 @@
...
+                        <i class="fas fa-ellipsis-v card-icons"></i>
+                         
+                         
+                     </footer>
+                    
+                 
+                 </div>
+                
+            </div>
+
```
Comment (Aditi0505) [inline components/cards/cards.html:95]: Can remove extra spacing and/ or use a code formatter.

File: components/cards/cards.html [hunk 11]
```diff
@@ -0,0 +1,277 @@
...
+            <div class="type-wrapper btn-wrapper">
+                <div class="component-type">
+                    <span class="type-title"> Horizontal Card </span>
+                </div> 
+                <!-- <div class="about-type">
+                    <p class="about">There are multiple sizes for square avatars as well. </p> 
+                </div> -->
```
Comment (Aditi0505) [inline components/cards/cards.html:102]: Can remove commented code.

File: components/cards/cards.html [hunk 12]
```diff
@@ -0,0 +1,277 @@
...
+                <div class="basic-card horizontalCard">
+                    <img class="horizontalImage" src="https://m.media-amazon.com/images/I/718FX4+xmCL._AC_SS450_.jpg" alt="">
+                    
+                     <div class="card-title horizontalTitle">
+                         <h3 class="title ">Castrol</h3>
+                         <p class="author"> by Kurt Wagner</p>
+                     </div>
+                      
+                     <!-- <div class="content horizontalContent">Castrol POWER1 4T 10W-30 API SL Synthetic Engine Oil for Bikes (900ml) </div> -->
+
```
Comment (Aditi0505) [inline components/cards/cards.html:115]: Can remove commented code and extra spacing.

File: components/cards/cards.css [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+    top:4rem;
+    left:7rem;
+}
+
+.card-badge{
+    background-color:var(--attention);
+    padding:3px;
+    text-align:center;
+    border-radius:5px;
+    color:whitesmoke;
```
Comment (Aditi0505) [inline components/cards/cards.css:91]: Can use hex code here.

File: components/cards/cards.html [hunk 14]
```diff
@@ -0,0 +1,277 @@
...
+                
+            </div>
+
+            <div class="type-wrapper btn-wrapper">
+                <div class="component-type">
+                    <span class="type-title"> Badges on Card </span>
+                </div> 
+                <!-- <div class="about-type">
+                    <p class="about">There are multiple sizes for square avatars as well. </p> 
+                </div> -->
```
Comment (Aditi0505) [inline components/cards/cards.html:140]: Can remove commented code.

File: components/cards/cards.html [hunk 15]
```diff
@@ -0,0 +1,277 @@
...
+                 </div>
+
+
+                 <div class="type-wrapper btn-wrapper">
+                    <div class="component-type">
+                        <span class="type-title"> Dismiss Icon on Card </span>
+                    </div> 
+                    <!-- <div class="about-type">
+                        <p class="about">There are multiple sizes for square avatars as well. </p> 
+                    </div> -->
```
Comment (Aditi0505) [inline components/cards/cards.html:178]: Same as above.

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. These are pretty much smalls changes to make. If you have bandwidth, you can make these changes now else can cover this later. 

Good use of sematic html like nav, header, main and aside! Good efforts!

**Pull request merged**